andela-Bgathu/GDocs
# 142905317 add documentation

### What does this PR do?
Adds documentation 
### how can this be manually tested
Check in the repo to see if there exists a `gdocs.apib`  file which contains the documentation and can be opened at Apiary
